packages/client/src/license/TermsAndConditions.ts

```diff
@@ -0,0 +1,37 @@
+import { UUID } from "@logion/node-api";
+
+export type TermsAndConditionsElementType = "logion_classification" | "specific_license";
```

Comment:
Is it `specific_license` or `specific_terms_and_conditions`?

Reply:
According to [this doc](https://docs.google.com/document/d/1D9kYRUWj-sPCdgCPWr8kHXB8Ncn-yNapCeWc3VUY1mk/edit) this is well `specific_license`.